diff --git a/Cargo.toml b/Cargo.toml
index 1a8cf16..b2a7ffa 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -13,7 +13,7 @@ description = "A simple Jellyfin Client written by GTK4-RS"
 license = "GPL-3.0"
 
 [dependencies]
-gtk = { version = "0.11", package = "gtk4", features = ["v4_22"] }
+gtk = { version = "0.11", package = "gtk4", features = ["v4_20"] }
 adw = { version = "0.9", package = "libadwaita", features = ["v1_8"] }
 serde = { version = "1.0.228", features = ["derive"] }
 serde_json = "1.0"
